tests/e2e/utils/flows.js

```diff
@@ -127,6 +127,13 @@ export const shopperWCP = {
 		await page.waitForNavigation( {
 			waitUntil: 'networkidle0',
 		} );
+
+		if ( null !== page.$( '#wc-woocommerce_payments-payment-token-new' ) ) {
+			await expect( page ).toClick(
+				'#wc-woocommerce_payments-payment-token-new'
```

Comment:
**Nitpick:** Looks like we have a helper to set checkboxes, is it possible to use that here? https://github.com/Automattic/woocommerce-payments/commit/ec75f7e7459cc528d1065467e59378e1d67df86a#diff-f46eace437806cd948229e60720806a3180ff5edb8e0a0192de5f3a91b8e7843R30